packages/controllers/src/snaps/SnapController.ts

```diff
@@ -434,6 +494,11 @@ type SnapControllerArgs = {
    */
   idleTimeCheckInterval?: number;
 
+  /**
+   * A function that checks whether the specified snap and version is blocked.
+   */
+  checkBlockList?: CheckSnapBlockList;
```

Comment:
Please don't add new functions to the constructor as we are trying to remove these 😅

Reply:
We are trying to remove all function arguments to the constructor, if those functions are methods on other controllers. In this case, there's no other controller that houses the `checkBlockList` function, and we don't have a way of specifying actions that exist in the `MetaMaskController` and consuming them elsewhere, so I think this is the best we can do for now.

Reply:
I thought we agreed that we want the `MetaMaskController` to start registering actions instead? I.e. https://github.com/MetaMask/snaps-skunkworks/blob/f7c94a069581951b343f0d596400dffc2c689f4f/packages/controllers/src/snaps/SnapController.ts#L982-L986

Reply:
Frederik and I discussed this, and we determined that functions from the `MetaMaskController` are fine for the time being because (1) it works and will continue to work fine and (2) it would be quite onerous and not very beneficial to define actions in libraries only to actually implement them in the extension repository. Once all controllers are in one big, happy monorepo, we'll use actions even for the `MetaMaskController`.